package.json

```diff
@@ -1,22 +1,25 @@
 {
   "name": "tsheets-sdk",
-  "version": "0.1.3",
+  "version": "1.0.0-0",
```

Comment:
Is the `-0` necessary on the end of this for a particular reason? I have most commonly used plane `1.0.0` and reserved the `-` for alpha/beta such as `1.0.0-alpha`.